### PR TITLE
Track event processing length as metric

### PR DESCRIPTION
**Pull request by marc-casperlabs**
Missing part from the `quanta` branch.

### CODE REVIEW

**Review by EdHastingsCasperLabs — APPROVED** [review]

File: node/src/reactor.rs [hunk 1]
```diff
@@ -234,10 +237,42 @@ impl RunnerMetrics {
...
+                300_000.0,
+                400_000.0,
+                500_000.0,
+                600_000.0,
+                700_000.0,
+                800_000.0,
+                900_000.0,
+                1_000_000.0,
+                2_000_000.0,
+                5_000_000.0,
```
Comment (goral09) [inline node/src/reactor.rs:266]: Scala lib comes with some default buckets that can be overwritten with config. Are there default buckets for Rust lib but they're too big for us?

File: node/src/reactor.rs [hunk 2]
```diff
@@ -234,10 +237,42 @@ impl RunnerMetrics {
     /// Create and register new runner metrics.
     fn new(registry: &Registry) -> Result<Self, prometheus::Error> {
         let events = IntCounter::new("runner_events", "total event count")?;
+
+        // Create an event dispatch histogram, putting extra emphasis on the area between 1-10 us.
```
Comment (Fraser999) [inline node/src/reactor.rs:241]: That comment isn't accurate is it?  Looks more like we're fine-grained for 100 us to 1 ms.

File: node/src/reactor.rs (hunk 1 above)
Comment (Fraser999) [reply node/src/reactor.rs:266]: That sounds useful for us, but confusing for end users faced with an increasingly complex config file?

Maybe we need a `dev`, `debug` or `advanced` section of the config if we do take this approach?

Comment (goral09) [reply node/src/reactor.rs:266]: In Scala there was a separate file for this (`application.conf`). Node configuration was in its own file and chainspec in its own dir structure. It wasn't convention invented by us but rather used by the library authors (and not specifically Prometheus library but whole Java/Scala ecosystem).

Comment (marc-casperlabs) [reply node/src/reactor.rs:266]: Buckets can be configured, they are just an a vec of values passed in. I'd rather have some sensible defaults first though, and make them configurable when we need to.

File: node/src/reactor.rs (hunk 2 above)
Comment (marc-casperlabs) [reply node/src/reactor.rs:241]: You're right :(

The code is correct, the comment is wrong; that area was specifically the intended high-res section.

**Pull request merged**